DDSim/DDSim/DD4hepSimulation.py

```diff
@@ -76,6 +76,7 @@ def __init__(self):
     self._errorMessages = []
     self._dumpParameter = False
     self._dumpSteeringFile = False
+    self.enableDetailedHitsAndParticleInfo = False
```

Comment:
There is no argument parsing defined for this flag, so it is essentially a constant.
Should be better defined in the ParticleHandler helper
https://github.com/iLCSoft/lcgeo/blob/master/DDSim/DDSim/Helper/ParticleHandler.py
Then it will become a command line parameter automatically.
And add a docstring like 
```
self.enableDetailedHitsAndParticleInfo = False
self._enableDetailedHitsAndParticleInfo_HELP = "print out lots of info about MCHistory"
```